index.html

```diff
@@ -43,5 +43,108 @@ <h2>I'm a web developer</h2>
+                    <li><p>HTML</p></li>
+                    <li><p>Bootstrap</p></li>
+                    <li><p>Ruby</p></li>
+                </ul>
+                <button class="see-project2">See project</button>
+            </div>
+            <div class="grid3">
+                <h2>Profesional Art Printing Data</h2>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+                <ul>
+                    <li><p>HTML</p></li>
+                    <li><p>Bootstrap</p></li>
+                    <li><p>Ruby</p></li>
+                </ul>
+                <button class="see-project2">See project</button>
+            </div>
+             <div class="grid3">
+                <h2>Profesional Art Printing Data</h2>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+                <ul>
+                    <li><p>HTML</p></li>
+                    <li><p>Bootstrap</p></li>
+                    <li><p>Ruby</p></li>
+                </ul>
+                <button class="see-project2">See project</button>
+            </div>
+            <div class="grid3">
+                <h2>Profesional Art Printing Data</h2>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+                <ul>
+                    <li><p>HTML</p></li>
+                    <li><p>Bootstrap</p></li>
+                    <li><p>Ruby</p></li>
+                </ul>
+                <button class="see-project2">See project</button>
+            </div>
+        </div>
+    </section>
+    <section class="about-section">
+        <div class="about-me">
+            <h1>About me</h1>
+            <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+            <button class="get-resume">Get my resume</button>
+        </div>
+        <ul class="knowledge">
+                <li><h2><img src="./Normal_Button/language-rectangle.png"> Language</h2></li>
+                <li>Javascript</li>
+                <li>Ruby</li>
+                <li>HTML</li>
+                <li>CSS</li>
+                <li><h2><img src="Normal_Button/frameworks-rectangle.png">   Frameworks</h2></li>
+                <li>React.js</li>
+                <li>Ruby on Rails</li>
+                <li>RSpec</li>
+                <li>Capybara</li>
+                <li><h2><img src="Normal_Button/skills-circle.png"> Skills</h2></li>
+                <li>Database Managment</li>
+                <li>Version Control</li>
+                <li>CLI</li>
+                <li>API Design</li>
+                <li>Web Development</li>
+        </ul>
+    </section>
```

Comment:
- [ ] Kindly consider using a nested unordered list to markup your list of technologies. Note that 'languages', 'frameworks', and 'skills' are list items of the parent unordered list and 'html', 'css', and 'javascirpt' are list items under languages. THis is better for HTML semantics. Kindly use the markup below as a guide;
```html

<ul>
  <li>Languages
    <ul>
        <li>JavaScript</li>
        <li>HTML</li>
        <li>CSS</li>
    </ul>
  </li>
  <li>Frameworks
    <ul>
        <li>React</li>
        <li>Ruby on Rails</li>
        <li>Rspec</li>
    </ul>
  </li>
  <li>Skills
    <ul>
        <li>Version control</li>
    ...
    </ul>
  </li>
</ul>
```